templates/web/base/around/geolocate_link.html

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19.83 4.17c.17.17.218.429.12.65l-6.594 14.834a.583.583 0 01-1.069-.008L9.91 14.091l-5.557-2.38a.583.583 0 01-.006-1.068L19.18 4.05a.583.583 0 01.648.12h.001z" fill="#000" fill-rule="nonzero"/></svg> 
```

Comment:
Seems a bit funky that there’s so much whitespace around the "arrow" in this image. The arrow itself is about 17px square, but the canvas is 24px. If we leave the SVG at its default 24px height in the button, it ends up causing odd vertical spacing above and below the button label text.

I’m thinking maybe either the arrow should be bigger, so it extends to the edges of the canvas (or thereabouts) or the canvas should be smaller.